.travis.yml

```diff
@@ -11,20 +13,21 @@ branches:
 language: generic
 
 env:
-  - NPM_TAG=latest IMAGE_NAME=theia-full NODE_VERSION=10.15.3
-  - NPM_TAG=next IMAGE_NAME=theia-full NODE_VERSION=10.15.3
-  - NPM_TAG=latest IMAGE_NAME=theia NODE_VERSION=10
-  - NPM_TAG=next IMAGE_NAME=theia NODE_VERSION=10
-  - NPM_TAG=latest IMAGE_NAME=theia-java NODE_VERSION=10
-  - NPM_TAG=next IMAGE_NAME=theia-java NODE_VERSION=10
-  - NPM_TAG=latest IMAGE_NAME=theia-go NODE_VERSION=10
-  - NPM_TAG=next IMAGE_NAME=theia-go NODE_VERSION=10
-  - NPM_TAG=latest IMAGE_NAME=yangster NODE_VERSION=10
-  - NPM_TAG=next IMAGE_NAME=yangster NODE_VERSION=10
-  - NPM_TAG=latest IMAGE_NAME=theia-python NODE_VERSION=10
-  - NPM_TAG=next IMAGE_NAME=theia-python NODE_VERSION=10
-  - NPM_TAG=latest IMAGE_NAME=theia-swift NODE_VERSION=10
-  - NPM_TAG=next IMAGE_NAME=theia-swift NODE_VERSION=10
+#  - NPM_TAG=latest IMAGE_NAME=theia-full NODE_VERSION=10.15.3
```

Comment:
Is there a reason for commenting these out?

Reply:
This temporary. I wanted to test my pull request with Travis, without running the tests for the other images.

Reply:
Reverted the changes